server/constants/messages.go

```diff
@@ -24,53 +24,56 @@ const (
 	EventTypeRequired               = "event type is required"
 	ServiceTypeRequired             = "service type is required"
 	ChannelIDRequired               = "channel ID is required"
+	WebhookSecretRequired           = "webhook secret is required"
 	MMUserIDRequired                = "mattermsot user ID is required"
 	EmptyAzureDevopsAPIBaseURLError = "azure devops API base URL should not be empty"
 	EmptyAzureDevopsOAuthAppIDError = "azure devops OAuth app id should not be empty"
 	// #nosec G101 -- This is a false positive. The below line is not a hardcoded credential
 	EmptyAzureDevopsOAuthClientSecretError = "azure devops OAuth client secret should not be empty"
 	EmptyEncryptionSecretError             = "encryption secret should not be empty"
+	WebhookSecretError                     = "webhook secret should not be empty"
```

Comment:
```suggestion
	EmptyWebhookSecretError                     = "webhook secret should not be empty"
```
Just for consistancy